src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml.cs

```diff
@@ -6,7 +6,11 @@
 using Dynamo.UI.Commands;
 using Dynamo.ViewModels;
 using Dynamo.Wpf.ViewModels.Core;
-using Dynamo.Core;
+#if NETFRAMEWORK
+using NotificationObject = Microsoft.Practices.Prism.ViewModel.NotificationObject;
+#else
+using NotificationObject = Dynamo.Core.NotificationObject;
```

Comment:
so just curious... were we even using this type today in net48?

Reply:
NotificationObject is used in net48
see here https://github.com/DynamoDS/Dynamo/blob/0323558faa1634097e3d3cae0e2ad0cf48cd3bac/src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml.cs#L232

Reply:
sorry @pinzart90 I should be more clear. Are we using `Dynamo.Core.NotificationObject` in net48? Or is that type you linked a subclass of the Microsoft Type? Or am I misunderstanding and that is a new type only in the new prism?

Reply:
Dynamo.Core.NotificationObject is used in net48. 
Dynamo.Core.NotificationObject and Microsoft.Practices.Prism.ViewModel.NotificationObject are separate implementations and they serve the same purpose

Reply:
Still confused. When is which used in net48? Do we need both? What about in net6? Do we only need Dynamo.Core.NotificationObject in net6?

Reply:
yeah, also still confused. If Dynamo.Core.NotificationObject is used in net48 - why are you doing this:
```
#if NETFRAMEWORK
using NotificationObject = Microsoft.Practices.Prism.ViewModel.NotificationObject;
```

Reply:
in net48:
Both Microsoft.Practices.Prism.ViewModel.NotificationObject and Dynamo.Core.NotificationObject are used. 
in net6:
NotificationObject is no longer part of Prism.Core. We switched to Dynamo.Core.NotificationObject

I wanted to revert all changes in net48 because some of the Microsoft.Practices.Prism types (including NotificationObject) bubble up to the public API. We cannot be sure how these types are used so better safe than sorry.
For example dynamo revit uses some of these types and expects that dynamo will have the Microsoft.Practices.Prism.dll